### ISSUE TITLE
Please help us for Tutorial wiki page update

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#768]
Hello folks,

The tutorial wiki page is written a long time ago and it really needs to be updated.
I think that the tutorial page is the entry point to the new users, so it is also important as well as the uftrace code update.

However, the project is almost managed by volunteers so we don't have enough time and resource to carefully read and update the tutorial all the time.

So we would like to invite anyone to help and update the Tutorial page. If you're a new comer, that's okay. Please read the tutorial and if you think that the contents are not understandable or incorrect, please let us know or you can also modify the document by yourself.

We would like to ask some help as follows:
- Fixing outdated or incorrect contents
- Better examples for uftrace features
- English and grammar correction
- Anything else that can be helpful

The wiki page is editable by anyone, so please help us to make the project better.
Thanks very much for your help in advance.

**Comment by coleblackman** [comment]
I updated the tutorial page, mostly fixing grammar issues and some formatting. A cursory look seems to indicate the content is still up to date, though if you have reason to think it' s not I'll go back through and execute each command with the sample programs to check. I also don't know whether command line flags have since been added/removed or not, but I think for a basic tutorial that's not a huge issue. 

If you'd like I will do the same with the main README

**Comment by namhyung** [comment]
Thanks a lot for your work!  It'd be great if you could check other pages like you mentioned!

**Comment by honggyukim** [comment]
@coleblackman Thanks very much for your help. Please go ahead and let us know if you have some questions.

**Comment by coleblackman** [comment]
No problem. I have continued working on the other wiki pages. The content has been very helpful in optimizing my own usage of uftrace. 

However, I can't quite figure out what the following snippet, taken from the first paragraph of the "Filters" wiki page, is trying to say. Can you rephrase it?

> For this reason, it is generally better to apply filters on record while most of it works for other commands like replay, report, graph and dump too.

Feel free to assign the issue and I'll submit a pull request for the main README also. Thanks.

**Comment by honggyukim** [comment]
Sorry for the late response. the sentence you mentioned can be rephrased as

> For this reason, it is generally better to apply filters during record time but almost all the filters can also be applied for other commands such as replay, report, graph, script and dump too.

I modified the sentence in the wiki myself. So don't worry to change it again.

> Feel free to assign the issue and I'll submit a pull request for the main README also. Thanks.

If you find something to change in README, then it would also be helpful for us. Thanks.
